main_test.go

```diff
@@ -98,6 +98,39 @@ func TestNotificationServiceRoutes(t *testing.T) {
 	assert.Equal(t, 1, len(routes))
 	assert.Equal(t, routes[0].Message, `@notorious-bot: {"notification_alert_type":"test_alert","data":"foobar"}`)
 
+	// Works OK if value is an int instead of a string.
```

Comment:
nit: I prefer `t.Log("Works OK if value is an int instead of a string")` instead of a comment (same idea below), so that errors are easier to debug in the test output. I reserve comments in tests to explain code hackery in tests, and use `t.Log()` when describing the test's intent